app/assets/stylesheets/new.scss

```diff
@@ -0,0 +1,31 @@
+$h1-color:      #088da5;
+$border-color:  #000000;
+
+.border {
+  border: 2px solid $border-color;
+  margin: auto;
+  padding: 20px 30px;
+}
+
+body {
+  // background-image: url(https://static.pexels.com/photos/6989/city-love-rainbow-nyc.jpg);
+  background-size: cover;
```

Comment:
You shouldn't need this rule about background-size if you don't have a background image!